src/common/styles/colors.scss

```diff
@@ -233,7 +233,5 @@ $spinner-text: var(--spinner-text);
 // consistent colors
 $always-white: #ffffff;
 $always-black: #161616;
-$always-primary-text: rgba(0, 0, 0, 0.9);
```

Comment:
was this added as part of your onging work??? I think `colors.scss` holds the "official" color palette and thus, I'm not sure we should remove anything from here...

Reply:
this was added in a previous PR to fix the same issue, but then we decided to remove the whole Ada image, so we don't need it anymore.